Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -431,9 +431,9 @@ def scan_result_command(client: Client, args: Dict[str, Any], api_key) -> List[C
 
                     ip_indicator = Common.IP(
                         ip=qid_data['data']['indicator'],
-                        asn=qid_data['data']['properties']['geo']['asn'],
-                        geo_country=qid_data['data']['properties']['geo']['country'],
-                        port=qid_data['data']['attributes']['port'],
+                        asn=qid_data['data']['properties']['geo'].get('asn'),
+                        geo_country=qid_data['data']['properties']['geo'].get('country'),
+                        port=qid_data.get('data', {}).get('attributes', {}).get('port')
```

Comment:
```suggestion
                        port=qid_data.get('data', {}).get('attributes', {}).get('port'),
```
According to the errors log - this comma should fix all the problems